assignments/closure.js

```diff
@@ -1,11 +1,33 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+function myName() {
+  let name = "Nick";
 
+  function fullName() {
+    console.log(name + ' Kruger');
+  }
+  fullName();
+}
+
+myName();
 // ==== Challenge 2: Create a counter function ====
-const counter = () => {
+const counter = function () {
+  let number = 0;
+  return function () {
+    number++;
+    console.log(number);
+  }
   // Return a function that when invoked increments and returns a counter variable.
-};
+}
+
+const newCounter = counter();
```

Comment:
what is the purpose of newCounter?
look into the difference between ++i and i++ - that may be helpful here.